### DISCUSSION TITLE
Pagina Home assistant non  raggiungibile

### DISCUSSION THREAD

**Discussion by niork** [indomus/forum#2848] [HUB: Home Assistant]
Un saluto a tutti,
da domo-neofita ho provato a ripercorrere la guida di installazione per configurare Home Assistant core su Docker su Raspberry Pi5. Quello ho.
Al momento della verifica sul browser niente connessione, Ho provato con `ifconfig` ed inserito l'ip visualizzato appresso a `inet`, a smanettare il router Fritzbox 7530, niente.
questa la risposta del terminal al comando `docker logs home-assistant`:
[logs.txt](https://github.com/indomus/forum/files/13998682/logs.txt)

**Comment by snakuzzo** [comment]
Prova a dare un'occhiata qui...

https://community.home-assistant.io/t/homeassistant-cannot-start-due-to-memory-allocation-failure-on-aarch64/557765/3

Questa cosa è indicata anche nella doc ufficiale...

https://www.home-assistant.io/installation/alternative#optimizations

**Comment by niork** [comment]
Grazie infinite, ho provato un bel pò a cercare sul forum, ma quello di InDomus non avevo ancora pensato a quello ufficiale di Home-assistant, Vado a cercare quello che mi serve...
Grazie

**Reply by snakuzzo** [reply]
La ricerca già te l''ho fatta io :) Devi solo aggiungere una env var quando crei il container

**Comment by niork** [comment]
Si, ho aggiunto `- DISABLE_JEMALLOC: true` sopra `- TZ=Europe/Rome` ,  identandolo sotto `environment: ` , nel file .yaml. Ho controllato nella pagina web yamllint la correttezza del file, ma continua al esserci lo stesso risultato al comando `docker logs home-assistant`.

**Reply by snakuzzo** [reply]
Ma hai ricreato il container ?

**Reply by snakuzzo** [reply]
Ma comunque hai sbagliato a definire l'environment... è scritot male.
Se usi compose (come immagino, visto quello che hai scritto)...

```yaml
    environment:
      - "TZ=Europe/Rome"
      - "DISABLE_JEMALLOC=true"
```

**Comment by niork** [comment]
Ho usato `docker compose down home-assistant` e poi `docker compose up -d home-assistant`.
per quanto riguarda il codice yaml la differenza e che io l'ho scritto a righe invertite... e non stringate ""

**Reply by snakuzzo** [reply]
1. Non è necessario fare il `down`. Quando modifichi il file compose ti basta ridare un `up -d` ...ed in automatico te lo ricrea con la nuova configurazione. Se fai il down butti giù tutti i container cancellandoli e cancellando pure la/le network associate (studia i comandi docker, pls)
2. Scrivi il codice come ho fatto io e riprova

**Comment by niork** [comment]
OK, finalmente! Grazie, ora la pagina si carica tramite l'ip del raspberry.

**Reply by snakuzzo** [reply]
Sì, però che cacchio...rispondi sotto l'ultima risposta!
Per sta cosa hai creato 4 thread diversi per la stessa discussione!

**Reply by niork** [reply]
oops, scusa me ne sono accorto adesso...
